packages/quantic/package.json

```diff
@@ -11,28 +11,32 @@
     "url": "git+https://github.com/coveo/ui-kit"
   },
   "scripts": {
-    "test": "npm run test:lint && npm run test:typings && npm run test:unit",
-    "test:lint": "eslint force-app/main/default/lwc/ --format junit -o reports/eslint.xml",
+    "test": "npm run build:staticresources && npm run lint:check:tests && npm run validate:types && npm run test:unit",
```

Comment:
`build:staticresources` is necessary for running tests and it runs fairly quickly, so I added it here and removed it from the CI. This makes it consistent with the rule I set, which is that `test` should work without doing anything first.

Eventually, with Nx, we can make `build:staticresources` a dependency of `test`, and it will only re-run it when necessary.